getting-started/dotnet/01.AI-SETUP.md

```diff
@@ -0,0 +1,57 @@
+# Getting started with bots and AI
+
+This guide is intended to help you get started with bots and AI. It is not intended to be a comprehensive guide, but rather a conglamerate of tips that most bots moving to AI will require.
+
+## AI Setup
+
+Once you have a basic bot setup, it is time to prepare the bot for AI usage.
+
+To use OpenAI, first create your OpenAI API key using the [OpenAI documentation](https://platform.openai.com/) and storing that key in your .env file:
```

Comment:
For C#, the keys are not stored in `.env` file but in `appsettings.Development.json` or `appsettings.json`.

Reply:
I noticed this as well in the samples, should we make this consistent across all the samples?

Reply:
thank you~ @aacebo yes, definitely.